### PR TITLE
update pointer to webpack for local tests

### PR DESCRIPTION
**Pull request by nolawi**
Updates webpack bundle location to match local build

### CODE REVIEW

**Comment by nolawi** [comment]
Local spec tests are passing for me am not sure what else could be missing. Marking this as [WIP] for now

File: client/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "build:test": "NODE_ENV=test yarn run build",
     "build:demo": "NODE_ENV=development yarn run build",
     "build:production": "NODE_ENV=production yarn run build",
-    "dev:clear-build": "rm ../app/assets/webpack/* || true",
+    "dev:clear-build": "rm ../app/assets/javascripts/* || true",
```
Comment (mdbenjam) [inline client/package.json]: hmmm maybe outputting this to the javascripts folder is a bad idea. I think we might still have a stats.js file in there. 🤔

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -231,7 +231,7 @@ def read_csv(klass, date_shift)
   # If it does, it will not execute ReactOnRails, since that slows down tests
   # Thus this will only run once (to initially compile assets) and not on
   # subsequent test runs
-  if Dir["#{::Rails.root}/app/assets/webpack/*"].empty?
+  if File.exist?("#{::Rails.root}/app/assets/javascripts/webpack-bundle.js")
```
Comment (mdbenjam) [inline spec/rails_helper.rb]: This should be if it doesn't exist, not if it does exist.

Comment (nolawi) [reply spec/rails_helper.rb]: duh that makes sense

File: client/package.json (hunk 1 above)
Comment (nolawi) [reply client/package.json]: i see, we can just remove `bundle` and sourcemaps instead of the whole folder. LMK if i should do that

Comment (mdbenjam) [reply client/package.json]: Hmmm yeah, I shouldn't have put generated js here as well as non-generated js...  Yeah it probably should just look for things with webpack in the name. Hopefully we can remove the stats.js file in the future.

**Pull request merged**